tests fail on i686

The testsuite fails on i686, as it seems `st.integers(0, 10)` returns int64 and np.dtype(int) is int32

```
=================================== FAILURES ===================================
________________ test_gets_the_correct_data_shape_for_just_rows ________________
[gw8] linux -- Python 3.6.2 /usr/bin/python
tests/pandas/test_data_frame.py:61: in test_gets_the_correct_data_shape_for_just_rows
    rows=st.fixed_dictionaries({'A': st.integers(1, 10), 'B': st.floats()}))
src/hypothesis/executors.py:58: in default_new_style_executor
    return function(data)
src/hypothesis/core.py:121: in run
    return test(*args, **kwargs)
tests/pandas/test_data_frame.py:61: in test_gets_the_correct_data_shape_for_just_rows
    rows=st.fixed_dictionaries({'A': st.integers(1, 10), 'B': st.floats()}))
src/hypothesis/core.py:526: in timed_test
    result = test(*args, **kwargs)
tests/pandas/test_data_frame.py:64: in test_gets_the_correct_data_shape_for_just_rows
    assert table['A'].dtype == np.dtype(int)
E   AssertionError: assert dtype('int64') == dtype('int32')
E    +  where dtype('int64') = Series([], Name: A, dtype: int64).dtype
E    +  and   dtype('int32') = <class 'numpy.dtype'>(int)
E    +    where <class 'numpy.dtype'> = np.dtype
---------------------------------- Hypothesis ----------------------------------
Falsifying example: test_gets_the_correct_data_shape_for_just_rows(table=Empty DataFrame
Columns: [A, B]
Index: [])
_______________________ test_will_use_dtype_of_elements ________________________
[gw9] linux -- Python 3.6.2 /usr/bin/python
tests/pandas/test_series.py:59: in test_will_use_dtype_of_elements
    def test_will_use_dtype_of_elements(s):
src/hypothesis/executors.py:58: in default_new_style_executor
    return function(data)
src/hypothesis/core.py:121: in run
    return test(*args, **kwargs)
tests/pandas/test_series.py:59: in test_will_use_dtype_of_elements
    def test_will_use_dtype_of_elements(s):
src/hypothesis/core.py:526: in timed_test
    result = test(*args, **kwargs)
tests/pandas/test_series.py:60: in test_will_use_dtype_of_elements
    assert s.dtype == np.dtype(int)
E   AssertionError: assert dtype('int64') == dtype('int32')
E    +  where dtype('int64') = Series([], dtype: int64).dtype
E    +  and   dtype('int32') = <class 'numpy.dtype'>(int)
E    +    where <class 'numpy.dtype'> = np.dtype
---------------------------------- Hypothesis ----------------------------------
Falsifying example: test_will_use_dtype_of_elements(s=Series([], dtype: int64))
```
This is on archlinux.
If you need more details, let me know.

--- a/tests/pandas/test_data_frame.py
+++ b/tests/pandas/test_data_frame.py
@@ -61,7 +61,7 @@ def test_can_specify_just_column_count(df):
     rows=st.fixed_dictionaries({'A': st.integers(1, 10), 'B': st.floats()}))
 )
 def test_gets_the_correct_data_shape_for_just_rows(table):
-    assert table['A'].dtype == np.dtype(int)
+    assert table['A'].dtype == np.dtype('int64')
     assert table['B'].dtype == np.dtype(float)
 
 

--- a/tests/pandas/test_series.py
+++ b/tests/pandas/test_series.py
@@ -57,7 +57,7 @@ def test_can_generate_integral_series(s):
 
 @given(pdst.series(elements=st.integers(0, 10)))
 def test_will_use_dtype_of_elements(s):
-    assert s.dtype == np.dtype(int)
+    assert s.dtype == np.dtype('int64')
 
 
 @given(pdst.series(elements=st.floats(allow_nan=False)))

